### ISSUE TITLE
Move Authenticator to phone

### ISSUE THREAD

**Issue by addee99** [Jessecar96/SteamDesktopAuthenticator#910]
Hey! I currently have a Steam account linked to my own number and using the Steam mobile app as authenticator. Im just wondering if I can add another account to my steam mobile app With a different number connected to the steamguard than my main account. And still have both account in the mobile app? If im not clear than tell me so I can try to explain better xD

**Comment by dyc3** [comment]
I don't see any reason why you shouldn't be able to.

**Comment by LORD47** [comment]
Yes, you can add and use multiple accounts on the same phone in the Steam app and use the authenticator for all the accounts.
And if your other account has already the Steam guard authenticator option enabled already (via SDA or any other authenticator) , you better move/transfer it to your Steam app to get only 2 days Trade Hold rather than 15 days.

**Comment by Allower** [comment]
Yes, there is no point what phone number you have linked to account. You can free to add any account to mobile phone app.

**Comment by DumbJoe** [comment]
Be careful, I got told if you use the same number for all your accounts and one gets a ban for whatever reasons, then all of them do so if you want to test something out to see if your account gets banned (hacking or cheating or whatever) or not without using your main account, I would advise using a different phone number for each and every account you have.

**Comment by bottleblue** [comment]
> , you better move/transfer it to your Steam app to get only 2 days Trade Hold 
?rather than 15 days.
>
How do I transfer it to avoid a 15 day wait?

**Comment by LORD47** [comment]
> > you better move/transfer it to your Steam app to get only 2 days Trade Hold
> > rather than 15 days.
> 
> How do I transfer it to avoid a 15 day wait?

Follow the steps mentioned on Steam support page section: [Can I transfer my authenticator to my new phone?](https://help.steampowered.com/en/faqs/view/7EFD-3CAE-64D3-1C31#transfer)

**Comment by bottleblue** [comment]
> Follow the steps mentioned on Steam support page section: [Can I transfer my authenticator to my new phone?](https://help.steampowered.com/en/faqs/view/7EFD-3CAE-64D3-1C31#transfer)

My authenticator is currently on Winauth. I didn't find a way to transfer the authenticator from Winauth to SDA. Should I transfer it to steam mobile and then to SDA? Do you know if that can be done without running into the 15 day wait?

**Comment by LORD47** [comment]
> I didn't find a way to transfer the authenticator from Winauth to SDA

I think there's a way to transfer directly from  Winauth to SDA, but I'm not really sure if the tool that allows that is safe to use.

> Should I transfer it to steam mobile and then to SDA? 

I believe that in order to do that you'd have to root your phone first, but I'm not seeing why you want to transfer to SDA in its current state, as the most secure option has always been to use the official Steam Guard authenticator unless you want to manage a lot of accounts and confirm mass trading and market listing (the Steam app support using multiple accounts but you can only confirm one by one), if that's the case and you're very familiar with [ASF](https://github.com/JustArchiNET/ArchiSteamFarm/releases) you can use its [2FA feature](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Two-factor-authentication) as an authenticator and it supports  [importing from Winauth](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Two-factor-authentication#winauth) and any [other authenticator](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Two-factor-authentication#import).

> Do you know if that can be done without running into the 15 day wait?

If you transfer to the Steam app you get 2 days Trade Hold while no Trade Hold at all when you use ASF 2FA and 15 days if you use something else.

**Comment by bottleblue** [comment]
> but I'm not seeing why you want to transfer to SDA in its current state
>

Because I'd rather use a standalone authenticator with a gui instead of general purpose one like ASF which doesn't have one and is not as easy to setup or use. How do I even use ASF to authenticate?

Is SDA still having issues after being updated? I see some people are having problems in the forum but I didn't think it was widespread.

**Comment by LORD47** [comment]
> Because I'd rather use a standalone authenticator with a gui 

But I was talking about using the Steam app instead of using the current version(s) of SDA

> instead of general purpose one like ASF which doesn't have one 

there's the [ASF-UI](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/IPC#asf-ui) for it that you can use from your browser.

> and is not as easy to setup or use. 

I get it, its features and the [configuration ](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Configuration) process are very overwhelming for first time users, but it's not that hard if you read its [Wiki](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/)

> Where do I go in ASF to authenticate?

after setting up the ASF 2FA (as I explained in my previous reply with Links) with your bot(s) (account(s)), just type the command `!2fa your_bot_name` and it would generate a token for you and if you want to confirm all trading/market listing you execute the command `!2faok your_bot_name ` and `!2fano your_bot_name` to reject and for more details check the [Commands](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Commands#commands-1) section on the ASF's Wiki
 
> Is SDA still having issues after being updated? I see some people are having problems in the forum but I didn't think it was widespread.

Well, the version that's supposed to fix the current issues is still in pre-release state and many people are facing a lot of errors

**Comment by bottleblue** [comment]
I'm not a first time ASF user but I only use it for the card farming. Setting it up just for that was confusing enough. 

I sometimes sell a lot of stuff on steam market so confirming one by one in steam app is annoying. I think I'll try SDA first and if that doesn't work I'll try ASF.

Thanks for the tips I appreciate it!

**Comment by bottleblue** [comment]
> 
> after setting up the ASF 2FA (as I explained in my previous reply with Links) with **your bot(s) (account(s))**, just type the command `!2fa your_bot_name` and it would generate a token for you and if you want to confirm all trading/market listing you execute the command `!2faok your_bot_name ` and `!2fano your_bot_name` to reject and for more details check the [Commands](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Commands#commands-1) section on the ASF's Wiki
>

When you say bot account does it have to be an actual bot account? You can use your main account to confirm trades also right?

**Comment by Hrxn** [comment]
Yes, of course you can

**Comment by bottleblue** [comment]
ok thx

**Comment by LORD47** [comment]
> When you say bot account does it have to be an actual bot account? 

no, that's just how accounts are referred to when using ASF and there's an option to make your main account as the Master account so you can use a command like **Loot** to send items (trading cards, ...) from your other accounts to your main one (the Master).

> You can use your main account to confirm trades also right?

Well, of course you can, you just need to enable the 2FA for it to be able to.

**Comment by bottleblue** [comment]
k I'm going to use ASF because SDA is still kind of a mess right now.
